libs/sdk-core/src/lib/db/search/search.ts

```diff
@@ -103,13 +103,14 @@ export const search = (
   return manageSearchRequest(nuclia, kbid, searchMethod);
 };
 
-export const catalog = (nuclia: INuclia, kbid: string, query: string, options?: SearchOptions) => {
+export const catalog = (nuclia: INuclia, kbid: string, query: string, options?: SearchOptions, useGet?: boolean) => {
```

Comment:
I don't understand: why did you create this `useGet` option while you're not using it anywhere in your PR 🤔

Reply:
@mpellerin42 if `/catalog` supports POST now, I think it is nice to offer this through the SDK (even if we do not use it in the dashboard, it could be used by external users)